pyrevolve/evolution/individual.py

```diff
@@ -33,7 +33,7 @@ def id(self):
         return _id
 
     def export_genotype(self, folder):
-        self.genotype.export_genotype(f'{folder}/genotypes/genotype_{self.phenotype.id}.txt')
+        self.genotype._body_genome.export_genotype(f'{folder}/genotypes/genotype_{self.phenotype.id}.txt') #Change
```

Comment:
Why is this necessary?

Reply:
maybe it isn't. I removed this and changed lsystem_neat_genotype

Reply:
thanks, like this now is compatible with the old l-system genome :)